packages/job-manager/src/runner/runner.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Message, Queue } from 'azure-services';
+import { JobManagerConfig, ServiceConfiguration } from 'common';
+import { inject, injectable } from 'inversify';
+import { Logger } from 'logger';
+import * as moment from 'moment';
+import { Batch } from '../batch/batch';
+import { JobTaskState } from '../batch/job-task';
+import { PoolMetrics } from '../batch/pool-metrics';
+
+@injectable()
+export class Runner {
```

Comment:
>Runner [](start = 13, length = 6)

name is confusing since we have a runner package

Reply:
will worker works?